### DISCUSSION TITLE
Github Education Providing Proof

### DISCUSSION THREAD

**Discussion by gragris** [community/community#133754] [GitHub Education]
### Hi everyone,

Hi everyone. Im trying to provide proof for github education. But im having a really hard time even if its a really easy process. First problem is, my student id has my student id in and validation date in the opposites sides. And there is option to only take 1 photo. So im not able to provide a date and my student id in the same photo. And i don't have a webcam so im using 3rd party software to use my phone as one. But i can only use video camera and when i try to take a photo of my academic calendar, it blurs out so much that its not counted as a proof. And also my academic calendar is turkish so im not even sure github will recognize it as a proof. There is not an option to upload documents. I really want to use co pilot and other tools but i can't afford them. I really want to get a confirmation so if anyone had the same issues please enlighten me.

**Comment by SohanKafle** [comment]
Hi [gragris](https://github.com/gragris),

If you're having trouble providing proof for GitHub Education, try these steps:
           **1. Combine Images:** Use a photo editing tool to combine both sides of your student ID into one image.
           **2. Clear Photos:** Use a scanner or a higher quality camera to take clear photos of your documents.
           **3. Translate Proof:** If your academic calendar is in Turkish, provide a translated version alongside the original.

If issues persist, contact [GitHub Education Support](https://support.github.com/contact/education).

**Reply by gragris** [reply]
hi @SohanKafle. Thank you for your fast response and interest. 
1.I can combine photos but since i will be taking photos of my computer screen, its probably wont be enough. My valid dates are on a bandrol which is not very visible.
2.I simply won't have access to a higher quality camera.

I tried to proceed with these steps and my application is not even getting processed. I have more than enough proof but there is not enough ways for me to provide them.

**Reply by SohanKafle** [reply]
[gragris](https://github.com/gragris)
you can also Consider these options too :

**1. Combine Photos:** Use photo editing software to combine both sides of your ID into one image.
**2. Seek Help:** Ask a friend with a better camera to help you take clearer photos.
**3. Contact Support:** Explain your situation to [GitHub Education Support](https://support.github.com/contact/education) and ask if there are alternative methods to provide your proof.

**Comment by samus-aran** [comment]
Hi @gragris 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

I would also suggest you perhaps try to submit the application via your mobile if taking a photo with it makes it easier.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
